NOTICE.txt

```diff
@@ -4680,3 +4680,30 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+---
+
+## Elastic
```

Comment:
I don't think we've added any parts of Elasticsearch into 3.10, that is coming in 3.11. @coreyhulen can you confirm?